lib/season_stat_team.rb

```diff
@@ -77,4 +88,18 @@ def least_accurate_team(season)
     end
     inaccurate_team[1][:team_name]
   end
+
+  def most_tackles(season)
+    team_tackles = create_team_data_by_season(season).max_by do |team, team_data|
+      team_data[:tackles]
+    end
+    team_tackles[1][:team_name]
+  end
+
+  def fewest_tackles(season)
+    team_tackles = create_team_data_by_season(season).min_by do |team, team_data|
+      team_data[:tackles]
+    end
+    team_tackles[1][:team_name]
+  end
```

Comment:
used max and min by respectively to grab most and fewest tackles

Reply:
I like the way this is set up. Very clean and neat